pinot-tools/src/main/java/com/linkedin/pinot/tools/PinotToolLauncher.java

```diff
@@ -35,6 +35,8 @@
   @Argument(handler = SubCommandHandler.class, metaVar = "<subCommand>")
   @SubCommands({
       @SubCommand(name = "UpdateSegmentState", impl = UpdateSegmentState.class),
+      @SubCommand(name = "PurgeSegmentsFromFiler", impl = FilerSegmentPurger.class),
```

Comment:
that was the initial idea but I thought it would be better to automate this by running it as part of the controller. Similar to retention manager, we can schedule this in the controller and it can automatically clean up segments after they are removed from Helix. We can be more conservative and check that the files are older than 7 days before deleting